config/prow/cluster/starter.yaml

```diff
@@ -146,14 +271,17 @@ spec:
         args:
         - --dry-run=false
         - --config-path=/etc/config/config.yaml
```

Comment:
Sorry it's not about fixing my deployment but don't we need `job-config-path` and the mounted jobs in hook?

Reply:
Ah I guess we dont need it because we use in-repo config?

Reply:
Its a single file that has both prow config and job config

Reply:
kk got it